src/runtime/runtime_nintendoswitch_heap.go

```diff
@@ -0,0 +1,28 @@
+// +build nintendoswitch
+
+// +build gc.conservative gc.leaking
+
+package runtime
+
+import "unsafe"
+
+const heapSize = 16 * 1024 * 1024
+
+//go:extern _stack_top
+var stackTopSymbol [0]byte
```

Comment:
Where is this defined?

Reply:
That's from devKitPro libc. I will put a comment there.